Assets/Scripts/Tools/MultiCamTool.cs

```diff
@@ -121,7 +121,7 @@ public class GifPreset
         [Header("Video Processing")]
         [SerializeField] private TextMeshPro m_VideoRecordTimer;
         [SerializeField] private GameObject m_VideoSavingRoot;
-        [SerializeField] private string m_VideoSavingText = "Loading Video...";
+        [SerializeField] private string m_VideoSavingText = "Finalizing Video...";
```

Comment:
Do these need to be localized?

Reply:
I'll deal with this in a separate PR following #429